src/Amazon.SQS.ExtendedClient/AmazonSQSExtendedClient.Net45.cs

```diff
@@ -94,11 +94,20 @@ public override ReceiveMessageResponse ReceiveMessage(ReceiveMessageRequest rece
                 MessageAttributeValue largePayloadAttributeValue;
                 if (message.MessageAttributes.TryGetValue(SQSExtendedClientConstants.RESERVED_ATTRIBUTE_NAME, out largePayloadAttributeValue))
                 {
-                    var messageS3Pointer = ReadMessageS3PointerFromJson(message.Body);
-                    var originalMessageBody = GetTextFromS3(messageS3Pointer.S3BucketName, messageS3Pointer.S3Key);
-                    message.Body = originalMessageBody;
-                    message.ReceiptHandle = EmbedS3PointerInReceiptHandle(message.ReceiptHandle, messageS3Pointer.S3BucketName, messageS3Pointer.S3Key);
-                    message.MessageAttributes.Remove(SQSExtendedClientConstants.RESERVED_ATTRIBUTE_NAME);
+                    try
+                    {
+                        var messageS3Pointer = ReadMessageS3PointerFromJson(message.Body);
+                        var originalMessageBody = GetTextFromS3(messageS3Pointer.S3BucketName, messageS3Pointer.S3Key);
+                        message.Body = originalMessageBody;
+                        message.ReceiptHandle = EmbedS3PointerInReceiptHandle(message.ReceiptHandle, messageS3Pointer.S3BucketName, messageS3Pointer.S3Key);
+                        message.MessageAttributes.Remove(SQSExtendedClientConstants.RESERVED_ATTRIBUTE_NAME);
+                    }
+                    catch (AmazonClientException amazonClientException)
```

Comment:
Shouldn't the same `try/catch` be used when the message is removed from the queue?
I mean, if one tries to delete non-existing s3 payload, will exception be raised? 
If so, then it would be nice to have the same exception handling logics there.